mcc/ttl/main.go

```diff
@@ -12,8 +12,10 @@ import (
 // GetResourceRecordSet returns a slice containing all responses for specified
 // query. It may issue more than one request as each returns a fixed amount of
 // entries at most.
-func GetResourceRecordSet(params *route53.ListResourceRecordSetsInput, svc *route53.Route53) (resource_record_set []*route53.ResourceRecordSet) {
-	for resp_is_truncated := true; resp_is_truncated; {
+func GetResourceRecordSet(
+	params *route53.ListResourceRecordSetsInput,
+	svc *route53.Route53) (resourceRecordSet []*route53.ResourceRecordSet) {
```

Comment:
Maybe break it further so the params and returns are not in the same line? Try using backslash '\' in case it works like C.